bin/strata-cli/src/cmd/faucet.rs

```diff
@@ -1,8 +1,11 @@
 use std::str::FromStr;
 
-use alloy::{primitives::Address as StrataAddress, providers::WalletProvider};
+// use alloy::{primitives::Address as StrataAddress, providers::WalletProvider};
```

Comment:
Why are we commenting this?
Do we need to remove the whole dep?
Or is this temporary? If yes, add a `FIXME` or `TODO` comment

If temporary
```suggestion
#[allow(dead_code)] // FIXME: add back when <FOO>
use alloy::{primitives::Address as StrataAddress, providers::WalletProvider};
```